release-notes/Code_Libraries/Skyline.DataMiner.DataSources.OpenConfig/Skyline.DataMiner.DataSources.OpenConfig.Gnmi_4.0.0.md

```diff
@@ -0,0 +1,13 @@
+---
+uid: Skyline.DataMiner.DataSources.OpenConfig.Gnmi_4.0.0
+---
+
+# Skyline.DataMiner.DataSources.OpenConfig.Gnmi 4.0.0
+
+#### Support for gNMI notifications of which the value is of type BytesVal, ProtoBytes, DecimalVal, FloatVal, or LeaflistVal [ID_38649]
+
+Previously, when the middleware received gNMI notifications with values of type *BytesVal* or *ProtoBytes*, these were flagged as unsupported. Now these will get processed and handed over to consumers of the middleware through a byte[]. It is then up to the consumer to process these in the expected way.
```

Comment:
```suggestion
Previously, when the middleware received gNMI notifications with values of type *BytesVal* or *ProtoBytes*, these were flagged as unsupported. Now these will get processed and handed over to consumers of the middleware through a `byte[]`. It is then up to the consumer to process these in the expected way.
```

byte[] could be marked as code or is this not typically done in release note documentation?

Reply:
Adjusted!